x-pack/plugins/security_solution/public/management/common/url_routing/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './utils';
```

Comment:
FYI: This directory (`url_routing`) will be refactored and consolidated into a folder named `routing` that will contain multiple files. I did not want to do that in this PR because then I would have to move over the existing `routing.ts` module and that would have pulled in several others due to path changes. I will create a follow up PR to this one.